crates/core/src/repofile/snapshotfile.rs

```diff
@@ -1193,12 +1193,15 @@ pub struct PathList(Vec<PathBuf>);
 
 impl Display for PathList {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        if !self.0.is_empty() {
-            write!(f, "{:?}", self.0[0])?;
-        }
-        for p in &self.0[1..] {
-            write!(f, ",{p:?}")?;
-        }
+        let string = self
+            .0
+            .iter()
+            .map(|p| p.to_string_lossy())
+            .collect::<Vec<_>>()
+            .join(",");
+
+        write!(f, "{string}")?;
+
         Ok(())
```

Comment:
Actually using `.format(",").fmt(f)` on the iterator is the better option (especially as `itertools` is already in scope)

Reply:
I changed this.